doc/source/dev/conduct/code_of_conduct.rst

```diff
@@ -0,0 +1,185 @@
+Though no list can hope to be comprehensive, we explicitly honour diversity in:
+age, culture, ethnicity, genotype, gender identity or expression, language,
+national origin, neurotype, phenotype, political beliefs, profession, race,
+religion, sexual orientation, socioeconomic status, subculture and technical
+ability.
+
+Though we welcome people fluent in all languages, SciPy development is
+conducted in English.
+
+Standards for behaviour in the SciPy community are detailed in the Code of
+Conduct above. We expect participants in our community to meet these standards
+in all their interactions and to help others to do so as well.
+
+
+Reporting Guidelines
+--------------------
+
+While this code of conduct should be adhered to by participants, we recognize
+that sometimes people may have a bad day, or be unaware of some of the
+guidelines in this code of conduct. When that happens, you may reply to them
+and point out this code of conduct. Such messages may be in public or in
+private, whatever is most appropriate. However, regardless of whether the
+message is public or not, it should still adhere to the relevant parts of this
+code of conduct; in particular, it should not be abusive or disrespectful.
+
+If you believe someone is violating this code of conduct, you may reply to them
+and point out this code of conduct. Such messages may be in public or in
+private, whatever is most appropriate. Assume good faith; it is more likely
+that participants are unaware of their bad behaviour than that they
+intentionally try to degrade the quality of the discussion. Should there be
+difficulties in dealing with the situation, you may report your compliance
+issues in confidence to either:
+
+- The SciPy Code of Conduct Committee, at conduct@TODO (preferred)
+  Members of this committee are TODO
+
+Or alternatively to:
+
+- Chair of the SciPy Steering Committee: Ralf Gommers
+- Executive Director of NumFOCUS: Leah Silen
+
+
+Incident reporting resolution & Code of Conduct enforcement
+-----------------------------------------------------------
+
+All complaints will be reviewed and investigated and will result in a response
+that is deemed necessary and appropriate to the circumstances. The SciPy Code
+of Conduct Committee, and the SciPy Steering Committee (if involved), are
+obligated to maintain confidentiality with regard to the reporter of an
+incident. 
+
+The process for acting on any received code of conduct violation report at a
+high level (for all but the most severe cases) will be:
+
+1. acknowledge report is received
+2. reasonable discussion/feedback
+3. mediation (if feedback didn't help)
+4. enforcement via transparent decision by the Code of Conduct Committee (if mediation failed)
+
+The committee will respond to any report as soon as possible, and at most
+within 72 hours.  More details on how reports are handled can be found in
+:ref:`CoC_reporting_manual`.
```

Comment:
Mention the reporting manual early on in the section, so that the reader knows there are further detail available, should they be curious.  Otherwise, you read this whole section thinking "kind of light on specific detail".

Reply:
good point, done